app/views/content/blog/my-first-week-as-a-trainee-teacher.md

```diff
@@ -0,0 +1,69 @@
+---
+title: My first week as a trainee teacher
+author: Cindy Ventura
+date: "2021-09-17"
+images:
+  cindy-ventura:
+    path: "media/images/content/blog/cindy-ventura.jpg"
+    alt: "A photograph of trainee teacher Cindy Ventura"
+description: |-
+  Cindy recently started the Postgraduate Diploma in Education (PGDE) teacher training course at Sheffield University. She kept a diary during her first week on the course and shares her experiences here.
+keywords:
+  - teacher training
+  - trainee teacher course
+  - university-led teacher training
+tags:
+  - becoming a teacher
+---
+
+$cindy-ventura$
+
+Cindy recently started the Postgraduate Diploma in Education (PGDE) teacher training course at Sheffield University. She kept a diary during her first week on the course and shares her experiences here.
+
+Throughout my time at university, I knew I wanted a career that would make a difference and give me opportunities for lifelong learning. I was interested in languages and worked as a language assistant in different schools in Sheffield, which developed my interest in teaching. I applied for the PGDE program at Sheffield University and since my interview in March, the time has passed so fast that I now find myself preparing my backpack for my first day.
```

Comment:
```suggestion
Throughout my time at university, I knew I wanted a career that would make a difference and give me opportunities for lifelong learning. I was interested in languages and worked as a language assistant in different schools in Sheffield, which developed my interest in teaching. I applied for the PGDE programme at Sheffield University and since my interview in March, the time has passed so fast that I now find myself preparing my backpack for my first day.
```